### PR TITLE
Release 3.5

### PR DESCRIPTION
**Pull request by PhilipVinc**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,23 @@
...
+### New features
+
+### Bug Fixes
+
+
+## NetKet 3.5 (☀️ 18 August 2022)
 
 [GitHub commits](https://github.com/netket/netket/compare/v3.4...master).
 
+This release adds support and needed functions to run TDVP for neural networks with real/non-holomorphic parameters, an HDF5 logger, a function to compute the local estimators of an observable for a set of samples.
```
Comment (femtobit) [inline CHANGELOG.md:18]:
```suggestion
This release adds support and needed functions to run TDVP for neural networks with real/non-holomorphic parameters, an experimental HDF5 logger, and an `MCState` method to compute the local estimators of an observable for a set of samples.
```

**Review by femtobit — COMMENTED** [review]
Looks good to me. One suggestion:

**Comment by PhilipVinc** [comment]
@femtobit done. I also addressed two deprecation warnings that show up in the next release of jax (0.3.17) since I noticed them..

**Review by femtobit — APPROVED** [review]
:+1:

**Pull request merged**